### PR TITLE
Update gpu test node image to the latest COS gpu-prebuild image that

### PR DESCRIPTION
**Pull request by jiayingz**
supports latest nvidia gpu driver.
Also a couple cleanups to remove unneeded feature setting and test.

### CODE REVIEW

**Comment by krzyzacy** [comment]
cc @Katharine 
/lgtm

File: testgrid/config.yaml [hunk 1]
```diff
@@ -312,10 +312,6 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-device-plugin-gpu-beta
   alert_stale_results_hours: 24
   num_failures_to_alert: 12 # Runs every 2h. Alert when it's been failing for a day.
-- name: ci-kubernetes-e2e-gce-gpu-stable2
```
Comment (krzyzacy) [inline testgrid/config.yaml]: there's another reference to `ci-kubernetes-e2e-gce-gpu-stable2` if you grep

Comment (jiayingz) [reply testgrid/config.yaml]: done

File: config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-gce.yaml [hunk 2]
```diff
@@ -2,10 +2,8 @@ presets:
 - labels:
     preset-ci-gce-device-plugin-gpu: "true"
   env:
-  - name: KUBE_FEATURE_GATES
-    value: DevicePlugins=true
   - name: KUBE_GCE_NODE_IMAGE
-    value: gke-1-8-4-gke-1-cos-stable-63-10032-71-0-p-v171208-pre-nvda-gpu
+    value: gke-1134-gke-rc5-cos-69-10895-138-0-v190320-pre-nvda-gpu
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-gce.yaml:6]: FYI @vpickard @SergeyKanzhelev
I just stumbled across this while working on https://github.com/kubernetes/k8s.io/issues/1095

I don't know if this is at risk of falling out of date at some point

Comment (vpickard) [reply config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-gce.yaml:6]: @jiayingz Can you provide some reference on the lifespan on this image? Any links would be very helpful. We have been addressing issues in sig-node e2e tests related to outdated images. Also, I'm wondering what triggered the update of this image? Was it to enable additional functionality with a newer image?

**Pull request merged**